sdv/data_processing/data_processor.py

```diff
@@ -103,6 +104,78 @@ def __init__(self, metadata, enforce_rounding=True, enforce_min_max_values=True,
         if self._primary_key:
             self._keys.append(self._primary_key)
 
+    def _get_grouped_columns(self):
+        """Get the columns that are part of a multi column transformer.
+
+        Returns:
+            list:
+                A list of columns that are part of a multi column transformer.
+        """
+        return [
+            col for col_tuple in self.grouped_columns_to_transformers for col in col_tuple
+        ]
+
+    def _check_import_address_transformers(self):
+        """Check that the address transformers can be imported."""
+        has_randomlocationgenerator = hasattr(rdt.transformers, 'RandomLocationGenerator')
+        has_regionalanonymizer = hasattr(rdt.transformers, 'RegionalAnonymizer')
+        if not has_randomlocationgenerator or not has_regionalanonymizer:
+            raise ImportError(
+                'You must have SDV Enterprise with the address add-on to use the address features'
+            )
+
+    def _get_columns_in_address_transformer(self):
+        """Get the columns that are part of an address transformer.
+
+        Returns:
+            list:
+                A list of columns that are part of the address transformers.
+        """
+        try:
+            self._check_import_address_transformers()
+            return [
+                item for col_tuple, transformer in self.grouped_columns_to_transformers.items()
+                if isinstance(
+                    transformer, (
+                        rdt.transformers.RandomLocationGenerator,
+                        rdt.transformers.RegionalAnonymizer
+                    )
+                ) for item in col_tuple
+            ]
+        except ImportError:
+            return []
+
+    def _get_address_transformer(self, anonymization_level):
+        """Get the address transformer.
+
+        Args:
+            anonymization_level (str):
+                The anonymization level for the address transformer.
+        """
+        locales = self._locales if self._locales else ['en_US']
+        self._check_import_address_transformers()
+        if anonymization_level == 'full':
+            return rdt.transformers.RandomLocationGenerator(locales=locales)
+        elif anonymization_level == 'street_address':
+            return rdt.transformers.RegionalAnonymizer(locales=locales)
```

Comment:
Can remove the `elif` and just `return rdt.transformers.RegionalAnonymizer(locales=locales)` or raise an error if none of the options